src/app/create/codebases/codebases.component.html

```diff
@@ -1,3 +1,5 @@
 <div id="codebases" class="container content">
-  <h1>TODO</h1>
+  <!-- can someone please show me how to make this be a relative url for the current component's directory?
+       I really just want to use 'codebases.png', but I've been unable to figure out how to do that -->
+  <img src='/app/create/codebases/codebases.png' style="width:100%;">
```

Comment:
@qodfathr as long as the image is in the same directory, you can just use the file name. 

img src='codebases.png' style="width:100%;"